PythonVisualizations/newQueue.py

```diff
@@ -0,0 +1,473 @@
+        #d = self.list[index]
+        #cell_coords = d.display_shape #self.cellCoords(index)
+        #cell_center = [] #self.cellCenter(index)
+        coords = canvas.coords(self.list[index].display_val)
+        x0 = coords[0]
+        x1 = coords[0]
+        y0 = coords[1] - CELL_SIZE * 3 // 2
+        y1 = coords[1] - CELL_SIZE * 5 // 10
+        if high:
+            y0 -= 20
+        arrow = canvas.create_line(
+            x0, y0, x1, y1, arrow="last", fill=self.VARIABLE_COLOR)
+        if name:
+            label = canvas.create_text(
+                x1 + 2, y0, text=name, anchor=SW,
+                font=self.VARIABLE_FONT, fill=self.VARIABLE_COLOR)
+        return (arrow, label) if name else (arrow,)
+
+    # ANIMATION METHODS
+    def speed(self, sleepTime):
+        return (sleepTime * (scaleDefault + 50)) / (scale.get() + 50)
+    
+    def set(self, index, val):
+        # reset the value of the Element at that index to val
+        self.list[index].val = val
+
+        # get the position of the displayed value
+        pos = canvas.coords(self.list[index].display_val)
+
+        # delete the displayed value and replace it with the updated value
+        canvas.delete(self.list[index].display_val)
+        self.list[index].display_val = canvas.create_text(pos[0], pos[1], text=str(val), font=('Helvetica', '20'))
+
+        # update window
+        window.update()  
+        
+        
+    # insert item at rear of queue   
+    def insertRear(self, val):
+        didInsert = False
+
+        # Fix to self.rear starting at 1
+        if self.nItems == 0:
+            self.rear = 0
+            self.front = self.rear # circular
+            
+            # create new cell and cell value display objects
+            # Start drawing new one at rear
+            cell = canvas.create_rectangle(ARRAY_X0+CELL_SIZE*self.rear, ARRAY_Y0, \
+                                           ARRAY_X0+CELL_SIZE*(self.rear+1), ARRAY_Y0 + CELL_SIZE, \
+                                           fill=Queue.colors[Queue.nextColor], outline='')
+            cell_val = canvas.create_text(ARRAY_X0+CELL_SIZE*self.rear + (CELL_SIZE / 2), \
+                                          ARRAY_Y0 + (CELL_SIZE / 2), text=val, font=('Helvetica', '20'))
+            
+            
+            #insert the item
+<<<<<<< Updated upstream:PythonVisualizations/newQueue.py
+            self.list[self.rear] = (Queue.Element(val, Queue.colors[Queue.nextColor], cell, cell_val))
+            
+            self.nItems += 1
+            
+            # increment nextColor
+            Queue.nextColor = (Queue.nextColor + 1) % len(Queue.colors)
+        
+            self.onOffButtons()
+            
+            # update window
+            window.update()
+            
+        
+=======
+            self.list[self.rear] = drawable(val, Queue.colors[Queue.nextColor], cell, cell_val)
+            didInsert = True
+
+>>>>>>> Stashed changes:PythonVisualizations/SimpleQueue.py
```

Comment:
This is an unresolved merge conflict.

Reply:
have resubmitted pull request with #94 does it work now?

Reply:
Yes, PR #94 fixed the merge conflicts and the changes to files that did not need changes.  I have reviewed PR #94 and we can switch to that one for the follow-up.  I will close this PR.